Makefile

```diff
@@ -48,18 +72,25 @@ venv: requirements.txt
 	pip install -r requirements.txt
 	touch venv
 
-# Generate the Dockerfile from a Jinja2 template.
-dockerfile: venv templates/Dockerfile.j2
-	jinja2 \
-	  -D elastic_version='$(ELASTIC_VERSION)' \
-	  -D staging_build_num='$(STAGING_BUILD_NUM)' \
-	  -D release_manager='$(RELEASE_MANAGER)' \
-	  templates/Dockerfile.j2 > build/kibana/Dockerfile
-
-# Generate docker-compose.yml from a Jinja2 template.
-docker-compose.yml: venv
-	jinja2 \
-	  -D version_tag='$(VERSION_TAG)' \
-	  templates/docker-compose.yml.j2 > docker-compose.yml
-
-.PHONY: build clean flake8 push pytest test dockerfile docker-compose.yml
+# Generate the Dockerfiles from Jinja2 templates.
+dockerfile: venv templates/Dockerfile*.j2
```

Comment:
While here `*` is used, later on only `templates/Dockerfile.j2` seems to be referenced; are there multiple Dockerfile jinja2 templates?